services/QuillLMS/app/models/subscription.rb

```diff
@@ -123,7 +122,6 @@ class RenewalNilStripeCustomer < StandardError; end
   scope :not_recurring, -> { where(recurring: false) }
   scope :not_stripe, -> { where(stripe_invoice_id: nil) }
   scope :started, -> { where("start_date <= ?", Date.current) }
-  scope :paid_with_card, -> { where.not(stripe_invoice_id: nil).or(where(payment_method: 'Credit Card')) }
```

Comment:
Was this scope breaking a test? I added this in yesterday -- I would still want this scope in here so let me know if there's a problem with it that I can fix in order to keep this scope here!